src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -137,6 +137,7 @@ export const BaseArtworkFilter: React.FC<
     offset?: number
     savedSearchProps?: SavedSearchAttributes
     enableCreateAlert?: boolean
+    defaultValues?: (string | null)[]
```

Comment:
**thoughts**: what do you think about the fact that only `string[]` is always expected here?